### ISSUE TITLE
Metrics Goes to Zero After Resume Training

### ISSUE THREAD

**Issue by yerol89** [ultralytics/ultralytics#18528]
### Search before asking

- [X] I have searched the Ultralytics YOLO [issues](https://github.com/ultralytics/ultralytics/issues) and found no similar bug report.

### Ultralytics YOLO Component

Train

### Bug

Hello,

Actually I don't have an error message but I want to report something that I couldn't figure out.  

I deal with Crack Detection on Windshields and thought that it would be a good approach to take advantage of YOLO Concrete Cracks Detection Dataset. And I found a model trained with this dataset at this link : https://huggingface.co/OpenSistemas/YOLOv8-crack-seg/tree/main/yolov8x/weights

Then I downloaded the last.pt file for tuning with my custom dataset(cracks on windshield that I labeled using LabelStudio and then turned to YOLO format). Here is the code I run:

```
model = YOLO('D:\Projects\YunusEr\WindShield_HF\FineTune_Crack_V4\pre_weight\last.pt') 
model.train(
    data='D:\Projects\YunusEr\WindShield_HF\FineTune_Crack_V4\crackDetection_YOLO.yaml', 
    epochs=200          
)
```

When I interrupt training process and then try to resume training, I have something weird. The first epoch after interruption, all the metrics returns zero(0).
[results.csv](https://github.com/user-attachments/files/18309746/results.csv)

For example, in the attacment(results.csv), I interrupted model in epoch 39. Until this epoch, all the metrics was improving epoch by epoch. But after the training of  40th epoch metrics turn into 0.  This is the code for resuming training process:

```
model = YOLO(r'MY_DIR\runs\segment\train\weights\last.pt')
model.train(resume=True)
```

I also experienced this problem in an another model. So this is the second time I face this issue.

Thanks for your helps in advance.

 

### Environment

```
Ultralytics 8.3.54  Python-3.10.7 torch-2.2.1+cpu CPU (Intel Xeon Gold 6348 2.60GHz)
Setup complete  (16 CPUs, 128.0 GB RAM, 266.5/1024.0 GB disk)

OS                  Windows-10-10.0.17763-SP0
Environment         Windows
Python              3.10.7
Install             pip
RAM                 128.00 GB
Disk                266.5/1024.0 GB
CPU                 Intel Xeon Gold 6348 2.60GHz
CPU count           16
GPU                 None
GPU count           None
CUDA                None``
```

### Minimal Reproducible Example

Can seen in attachment of BUG part

### Are you willing to submit a PR?

- [ ] Yes I'd like to help by submitting a PR!

**Comment by UltralyticsAssistant** [comment]
👋 Hello @yerol89, thank you for reporting this and for your detailed description! 🚀 We appreciate your effort to share the context, code snippets, and the resulting behavior.

For issues like this, we recommend first upgrading to the latest version of the `ultralytics` package to ensure you have all the latest fixes and improvements. Please install the latest version in a [Python>=3.8](https://www.python.org/) environment with [PyTorch>=1.8](https://pytorch.org/get-started/locally/) using the command:

```bash
pip install -U ultralytics
```

### 🐛 Bug Report:
To help us better debug the issue, could you provide a [minimum reproducible example](https://docs.ultralytics.com/help/minimum_reproducible_example/)? This step is crucial for reproducing and fixing the bug efficiently. Specifically:
- Provide the complete dataset configuration file and any relevant dataset details in `crackDetection_YOLO.yaml`.
- Clarify if modifications were made to the `last.pt` weights or YOLO model architecture.
- Share additional logs from the training process (if possible) or any error messages.

### 🔧 Troubleshooting:
Additionally, please verify the following:
1. Did you prepare your data correctly according to the [Ultralytics Data Format](https://docs.ultralytics.com/datasets/)?
2. Are you following the [Tips for Best Training Results](https://docs.ultralytics.com/guides/model-training-tips/) during your fine-tuning process?

### 💡 Resources:
For more debugging and development tips, you can visit:
- The [Documentation](https://docs.ultralytics.com) for examples and FAQs.
- Our [Minimum Reproducible Example Guide](https://docs.ultralytics.com/help/minimum_reproducible_example/) for preparing a concise reproduction scenario.

### 💬 Community and Support:
If you’d like to engage further with the community or discuss this issue, consider joining us on:
- [Discord for real-time chat](https://discord.com/invite/ultralytics) 🎧
- [Discourse for discussions](https://community.ultralytics.com)
- [Subreddit for broader topics](https://reddit.com/r/Ultralytics)

### Verified Environments:
YOLO works seamlessly in multiple validated environments. If you want a simple way to troubleshoot:
- 🟢 Try replicating the issue in free GPU environments like:
  <a href="https://console.paperspace.com/github/ultralytics/ultralytics"><img src="https://assets.paperspace.io/img/gradient-badge.svg" alt="Run on Gradient"/></a> 
  <a href="https://colab.research.google.com/github/ultralytics/ultralytics/blob/main/examples/tutorial.ipynb"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"></a> 
  <a href="https://www.kaggle.com/models/ultralytics/yolo11"><img src="https://kaggle.com/static/images/open-in-kaggle.svg" alt="Open In Kaggle"></a>
- Or in cloud infrastructures via:
  - [Google Cloud](https://docs.ultralytics.com/yolov5/environments/google_cloud_quickstart_tutorial/)
  - [AWS Quickstart](https://docs.ultralytics.com/yolov5/environments/aws_quickstart_tutorial/)
  - Docker: [Guide](https://docs.ultralytics.com/yolov5/environments/docker_image_quickstart_tutorial/) <a href="https://hub.docker.com/r/ultralytics/ultralytics"><img src="https://img.shields.io/docker/pulls/ultralytics/ultralytics?logo=docker" alt="Docker Pulls"></a>

### CI Status:
<a href="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yaml?query=event%3Aschedule"><img src="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yaml/badge.svg" alt="Ultralytics CI"></a> The current CI status badge shows if all our automated tests are passing.

Thank you again for reaching out! This is an automated response 🤖, but an Ultralytics engineer will review and assist you further soon. Your input helps us improve YOLO for everyone 🌟.

**Comment by Y-T-G** [comment]
Are you training on CPU?

**Comment by yerol89** [comment]
> Are you training on CPU?

Unfortunately yes. It takes days to train a single model, but I have to.

**Comment by Y-T-G** [comment]
Does it occur with all models?

**Comment by yerol89** [comment]
> Does it occur with all models?

Nowadays I only work on this model, so I can't make a comparison with other YOLO models. 

But this started to happen after I updated ultralytics. It was running without an issue(even the training after interruption), before the update

**Comment by glenn-jocher** [comment]
Thank you for the clarification. This issue might be related to changes introduced in the update. I recommend rolling back to the previous version of Ultralytics that worked for you (`pip install ultralytics==<version>`) and testing again. If the issue persists, please let us know so we can investigate further.

**Comment by yerol89** [comment]
> Thank you for the clarification. This issue might be related to changes introduced in the update. I recommend rolling back to the previous version of Ultralytics that worked for you (`pip install ultralytics==<version>`) and testing again. If the issue persists, please let us know so we can investigate further.

Thanks for your reply. I will try this and let you know as soon as possible. Thanks for the support again.

**Comment by glenn-jocher** [comment]
You're welcome! Take your time to test it, and feel free to share the results or any further issues you encounter. The community and the Ultralytics team are here to help.
